first draft IN mapping

AudreyAltman opened the pull request:
This is a first pass at the Indiana mapping.

moltude commented on src/main/scala/dpla/ingestion3/mappers/providers/InMapping.scala:
```diff
@@ -0,0 +1,140 @@
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "provenance")
+      .map(nameOnlyAgent)
+
+  // TODO: In ingest1, the following are also mapped to edmRights:
```
@AudreyAltman yes, I think that is a valid assumption

moltude commented on src/main/scala/dpla/ingestion3/mappers/providers/InMapping.scala:
```diff
@@ -0,0 +1,140 @@
+      .flatMap(_.splitAtDelimiter(";"))
+
+  // TODO: This is in the crosswalk but not in ingest1. It is also identical to format mapping.  Keep?
+  override def genre(data: Document[NodeSeq]): ZeroToMany[SkosConcept] =
```
genre is a deprecated field in MAPv5 so this is going to be dropped in the near term anyway. I'm 50/50 on keeping it [tilt towards removing] as I do not believe `genre` is made available in the API.

moltude commented on src/main/scala/dpla/ingestion3/mappers/providers/InMapping.scala:
```diff
@@ -0,0 +1,140 @@
+  //  Ingest1 has the following enrichments that may indicate the need for some logic here:
+  //  "/enrich-type?default=image" ?
+  //  "/enrich-type?send_rejects_to_format=true"
+  override def `type`(data: Document[NodeSeq]): ZeroToMany[String] =
```
good catch. The solution might be to map the type field to format and then apply a format allowFilter on the type values. 

Not sure if we want to apply a default type value though...that seems awkward.

AudreyAltman commented:
@moltude I've addressed your comments here: https://github.com/dpla/ingestion3/pull/290/commits/f3dd5d98507f63da6ff5ccc8e3e88a236466ab3b Could you review them and make sure it all looks good?

AudreyAltman commented:
@mdellabitta strip trailing semicolons from `edmRights` is implemented here: https://github.com/dpla/ingestion3/pull/290/commits/febb65dbac3aee7acb46e2ecfb0b652e17c6476c
